### ISSUE TITLE
Recent tabs menu option causes crash

### ISSUE THREAD

**Issue by TheFlatEric** [bromite/bromite#2190]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

103.0.5060.121

### Device architecture

arm64-v8a

### Android version

12.0

### Device model

Pixel 3a

### Changed flags

No flags changed

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

[4eb0b8f6-b8bc-4bbd-87b0-0a3eefe57d39.zip](https://github.com/bromite/bromite/files/9098641/4eb0b8f6-b8bc-4bbd-87b0-0a3eefe57d39.zip)

### Describe the bug

Bromite crashes when selecting the recent tabs menu item.

### Steps to reproduce the bug

1. Open Bromite
2. Select menu
3. Click Recent Tabs
4. Bromite crashes

### Expected behavior

The list of recent tabs is displayed

**Comment by uazo** [comment]
this is the stack trace:
```
java.lang.IllegalStateException: AccountManagerFacade is not yet initialized!
        at org.chromium.components.signin.AccountManagerFacadeProvider.getInstance(AccountManagerFacadeProvider.java:81)
        at org.chromium.chrome.browser.ntp.RecentTabsManager.<init>(RecentTabsManager.java:105)
        at org.chromium.chrome.browser.native_page.NativePageFactory$NativePageBuilder.buildRecentTabsPage(NativePageFactory.java:210)
        at org.chromium.chrome.browser.native_page.NativePageFactory.createNativePageForURL(NativePageFactory.java:275)
        at org.chromium.chrome.browser.native_page.NativePageFactory.createNativePage(NativePageFactory.java:247)
        at org.chromium.chrome.browser.TabbedModeTabDelegateFactory.createNativePage(TabbedModeTabDelegateFactory.java:161)
        at org.chromium.chrome.browser.tab.TabImpl.maybeShowNativePage(TabImpl.java:1213)
        at org.chromium.chrome.browser.tab.TabImpl.loadUrl(TabImpl.java:498)
        at org.chromium.chrome.browser.ChromeTabbedActivity.onMenuOrKeyboardAction(ChromeTabbedActivity.java:2092)
        at org.chromium.chrome.browser.app.ChromeActivity.onOptionsItemSelected(ChromeActivity.java:1661)
        at org.chromium.chrome.browser.ui.appmenu.AppMenuHandlerImpl.onOptionsItemSelected(AppMenuHandlerImpl.java:307)
        at org.chromium.chrome.browser.ui.appmenu.AppMenu.onItemClick(AppMenu.java:408)
        at org.chromium.chrome.browser.ui.appmenu.AppMenuItemViewBinder.lambda$bindStandardItem$0(AppMenuItemViewBinder.java:76)
        at org.chromium.chrome.browser.ui.appmenu.AppMenuItemViewBinder$$ExternalSyntheticLambda3.onClick(Unknown Source:2)
        at android.view.View.performClick(View.java:6597)

```

**Comment by csagan5** [comment]
Fixed in `103.0.5060.126`.
